src/pages/product-detail.tsx

```diff
@@ -0,0 +1,19 @@
+import Footer from '../components/Footer'
+import Navbar from '../components/Navbar'
+import ProductView from '../components/ProductView'
+
+const ProductDetail = () => (
```

Comment:
For the path, would be better to use `Product` for both path and class name. For plural of product we use `ProductList` and `Product` for singular form.

Reply:
Don't forget to name files under `/pages` in lower case.

Reply:
So, you mean the name of this page should be `Product`, right? which import `ProductView` component. FYI, we use the name `Products` for the page of (old) `ProductList`. This is not going to make confusion, right? 🤨